Unable to run on android

Version Used:

  stream_video_flutter: ^0.10.1
  stream_video: ^0.10.1
  stream_video_push_notification: ^0.10.1


Here is the log:

Resolving dependencies...
Downloading packages...
  _fe_analyzer_shared 76.0.0 (86.0.0 available)
  _flutterfire_internals 1.3.59 (1.3.60 available)
  analyzer 6.11.0 (8.0.0 available)
  build 2.5.4 (3.0.0 available)
  build_resolvers 2.5.4 (3.0.0 available)
  build_runner 2.5.4 (2.6.0 available)
  build_runner_core 9.1.2 (9.2.0 available)
  characters 1.4.0 (1.4.1 available)
  cloud_firestore 5.6.12 (6.0.0 available)
  cloud_firestore_platform_interface 6.6.12 (7.0.0 available)
  cloud_firestore_web 4.4.12 (5.0.0 available)
  cloud_functions 5.6.2 (6.0.0 available)
  cloud_functions_platform_interface 5.8.2 (5.8.3 available)
  cloud_functions_web 4.11.5 (5.0.0 available)
  dart_style 2.3.8 (3.1.1 available)
  dart_webrtc 1.5.3+hotfix.3 (1.5.3+hotfix.4 available)
  file_picker 10.2.0 (10.2.1 available)
  firebase_analytics 11.6.0 (12.0.0 available)
  firebase_analytics_platform_interface 4.4.3 (5.0.0 available)
  firebase_analytics_web 0.5.10+16 (0.6.0 available)
  firebase_auth 5.7.0 (6.0.0 available)
  firebase_auth_platform_interface 7.7.3 (8.0.0 available)
  firebase_auth_web 5.15.3 (6.0.0 available)
  firebase_core 3.15.2 (4.0.0 available)
  firebase_core_web 2.24.1 (3.0.0 available)
  firebase_messaging 15.2.10 (16.0.0 available)
  firebase_messaging_platform_interface 4.6.10 (4.7.0 available)
  firebase_messaging_web 3.10.10 (4.0.0 available)
  firebase_remote_config 5.5.0 (6.0.0 available)
  firebase_remote_config_platform_interface 2.0.0 (2.0.1 available)
  firebase_remote_config_web 1.8.9 (1.8.10 available)
  firebase_storage 12.4.10 (13.0.0 available)
  firebase_storage_platform_interface 5.2.10 (5.2.11 available)
  firebase_storage_web 3.10.17 (3.10.18 available)
  flutter_callkit_incoming 2.5.2 (2.5.7 available)
  flutter_local_notifications 19.3.1 (19.4.0 available)
  freezed_annotation 2.4.4 (3.1.0 available)
  get_it 8.0.3 (8.2.0 available)
  go_router 16.0.0 (16.1.0 available)
  google_fonts 6.2.1 (6.3.0 available)
! intl 0.20.2 (overridden)
  intl_utils 2.8.8 (2.8.11 available)
  leak_tracker 10.0.9 (11.0.1 available)
  leak_tracker_flutter_testing 3.0.9 (3.0.10 available)
  leak_tracker_testing 3.0.1 (3.0.2 available)
  material_color_utilities 0.11.1 (0.13.0 available)
  meta 1.16.0 (1.17.0 available)
  petitparser 6.1.0 (7.0.0 available)
  purchases_flutter 8.10.6 (9.1.0 available)
! record_linux 1.1.1 (overridden)
  source_gen 1.5.0 (3.0.0 available)
  source_helper 1.3.5 (1.3.6 available)
  sqflite_common 2.5.5 (2.5.6 available)
  stream_chat 9.14.0 (9.15.0 available)
  stream_chat_flutter 9.14.0 (9.15.0 available)
  stream_chat_flutter_core 9.14.0 (9.15.0 available)
> stream_video 0.10.1 (was 0.9.6)
> stream_video_flutter 0.10.1 (was 0.9.6)
> stream_video_push_notification 0.10.1 (was 0.9.6)
  test_api 0.7.4 (0.7.7 available)
  vector_math 2.1.4 (2.2.0 available)
  video_player_android 2.8.7 (2.8.10 available)
  video_player_avfoundation 2.8.0 (2.8.2 available)
  vm_service 15.0.0 (15.0.2 available)
  webview_flutter_platform_interface 2.13.1 (2.14.0 available)
  xml 6.5.0 (6.6.0 available)
Changed 3 dependencies!
61 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
"es": 18 untranslated message(s).
To see a detailed report, use the untranslated-messages-file 
option in the l10n.yaml file:
untranslated-messages-file: desiredFileName.txt
<other option>: <other selection> 


This will generate a JSON format file containing all messages that 
need to be translated.
Launching lib/main_development.dart on sdk gphone64 arm64 in debug mode...
Running Gradle task 'assembleDevelopmentDebug'...
e: file:///Users/codeable/.pub-cache/hosted/pub.dev/stream_video_flutter-0.10.1/android/src/main/kotlin/io/getstream/video/flutter/stream_video_flutter/service/notification/image/CircleTransform.kt:19:54 Argument type mismatch: actual type is 'Bitmap.Config?', but 'Bitmap.Config' was expected.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stream_video_flutter:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 32s
Error: Gradle task assembleDevelopmentDebug failed with exit code 1

--- a/packages/stream_video_flutter/CHANGELOG.md
+++ b/packages/stream_video_flutter/CHANGELOG.md
@@ -9,6 +9,9 @@
 🔄 Dependency updates
 * Updated `flutter_callkit_incoming` dependency to the latests (2.5.5) version. That version contains Android 14 compatibility fixes for ringing notifications and lock screen handling.
 
+🐞 Fixed
+* (Android) CircleTransform Argument type mismatch on Bitmap.Config?
+
 ## 0.10.1
 
 🐞 Fixed

--- a/packages/stream_video_flutter/android/src/main/kotlin/io/getstream/video/flutter/stream_video_flutter/service/notification/image/CircleTransform.kt
+++ b/packages/stream_video_flutter/android/src/main/kotlin/io/getstream/video/flutter/stream_video_flutter/service/notification/image/CircleTransform.kt
@@ -16,7 +16,8 @@ class CircleTransform : Transformation {
         if (squaredBitmap != source) {
             source.recycle()
         }
-        val bitmap = Bitmap.createBitmap(size, size, source.config)
+        val config = source.config ?: Bitmap.Config.ARGB_8888
+        val bitmap = Bitmap.createBitmap(size, size, config)
         val canvas = Canvas(bitmap)
         val paint = Paint()
         val shader = BitmapShader(

